README.md

```diff
@@ -1 +1,133 @@
-# .github
\ No newline at end of file
+# Secure (This is a community driven project)
+
+`Secure` middleware for hertz framework.
+
+This repo is forked from [secure](https://github.com/gin-contrib/secure) and adapted for hertz.
+
+## Install
+
+```bash
+go get github.com/hertz-contrib/secure
+```
+
+### [Default example](example/default/main.go)
+
+Default configuration for users to set security configuration directly using secure middleware
+
+#### Function Signature
+
+```go
+func DefaultConfig() Config
+```
+
+#### Sample Code
+
+```go
+package main
```

Comment:
code need to format and readme need to improve

Reply:
GET